dev-python/types-psutil/types-psutil-5.9.5.16.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Gentoo Authors
```

Comment:
I think we want stabilization groups to combine them with the respective packages (there's already one for setuptools, so add it there).

Reply:
Done. (Could not find documentation but git grepped for setuptools and found where this was supposed to go.)

Reply:
Sorry, they're very new:
* https://marc.info/?l=gentoo-dev&m=168951940606443&w=2
* https://marc.info/?l=gentoo-dev&m=169263637219716&w=2